packages/react-dom/src/__tests__/ReactUpdates-test.js

```diff
@@ -1009,15 +1161,22 @@ describe('ReactUpdates', () => {
       }
     }
 
-    const parent = ReactTestUtils.renderIntoDocument(<Parent />);
+    const container = document.createElement('div');
+    const root = ReactDOMClient.createRoot(container);
+    await act(() => {
+      root.render(<Parent />);
+    });
+
     const child = parent.getChild();
-    ReactDOM.unstable_batchedUpdates(function () {
+    await act(() => {
       parent.forceUpdate();
       child.forceUpdate();
     });
+
+    expect.assertions(6);
```

Comment:
Added this, since there's wasn't anything confirming that the `expects` were actually running.